client/me/purchases/purchases-list/index.jsx

```diff
@@ -38,6 +46,36 @@ class PurchasesList extends Component {
 		return ! this.props.sites.length;
 	}
 
+	renderConciergeBanner() {
+		const { nextAppointment, scheduleId, hasAvailableConciergeSessions } = this.props;
+
+		if ( null === hasAvailableConciergeSessions ) {
+			return <ConciergeBanner bannerType="placeholder" />;
+		}
+
+		let bannerType;
+
+		if ( nextAppointment ) {
+			bannerType = 'upcomingAppointment';
+		} else if ( hasAvailableConciergeSessions ) {
+			if ( 1 === scheduleId ) {
+				bannerType = 'availableIncludedSession';
+			} else if ( 2 === scheduleId ) {
```

Comment:
It's not immediately clear what ID 1 & 2 mean here. Could we define some constants for them somewhere, e.g. https://github.com/Automattic/wp-calypso/blob/master/client/me/concierge/constants.js ?

Also, note that there are cases we intentionally override the schedule IDs from the backend upon testing so we won't really bother any HEs. In that case, `bannerType` would be `null` here. For now I'd suggest us to pick either one as the default, or introduce one specifically for testing if we don't want to confuse anyone.

Reply:
Fixed in [4d6a1d9](https://github.com/Automattic/wp-calypso/pull/34956/commits/4d6a1d996874a6dff8f16889b1bf560f27a168fe) and [7379361](https://github.com/Automattic/wp-calypso/pull/34956/commits/73793615cc0772b5e6c2c9d6712e3fbea946cf0d)